db/re/item_group_db.yml

```diff
@@ -46654,22 +46654,22 @@ Body:
             Item: Lian_Shoes
             Rate: 100
           - Index: 36
-            Item: aegis_300487
+            Item: S_Elena_Card
             Rate: 85
           - Index: 37
-            Item: aegis_300489
+            Item: S_Lookie_Card
             Rate: 85
           - Index: 38
-            Item: aegis_300491
+            Item: S_Lookie_Card
```

Comment:
Sorry I quote the wrong line/ID https://www.divine-pride.net/database/item/300491